sdk/helper/pluginutil/runner.go

```diff
@@ -35,12 +35,14 @@ type LookRunnerUtil interface {
 
 type Multiplexer interface {
 	ID() string
-	Conn() *grpc.ClientConn
-	MultiplexingSupport() bool
+	Protocol() grpc.ClientConnInterface
```

Comment:
Since we're still returning a `grpc.ClientConnInterface`, it might be clearer to call it `GRPCClientConn` or something along those lines.

Reply:
addressed in: https://github.com/hashicorp/vault/pull/13896/commits/7c1e08b56e41a2259c1550ea83d767f8a52bb6b5